suites/apis/linkGoogleAccountTest.js

```diff
@@ -39,3 +67,23 @@ Scenario('try to extend link when acct is not linked @reqGoogle', async (fence,
   const extendRes = await fence.do.extendGoogleLink(users.mainAcct);
   fence.ask.responsesEqual(extendRes, fence.props.resExtendNoGoogleAcctLinked);
 });
+
+Scenario('link an acct that is already linked to this user @reqGoogle', async (fence, users) => {
+  await fence.complete.linkGoogleAcctMocked(users.mainAcct);
+
+  // try to link mainAcct to the same account again
+  let linkRes = await fence.do.linkGoogleAcctMocked(users.mainAcct, users.mainAcct.googleCreds.email);
+  expect(linkRes.finalURL, 'Linking a google account twice should fail').to.not.exist;
```

Comment:
this should actually redirect with an `error` and `error_description`.. we should be able to check the `error` here... what is the response? Why is finalURL not getting filled out?

Reply:
should look like: `https://{{domain}}/?error_description=User+already+has+a+linked+Google+account.&error=g_acnt_link_error`

Reply:
this error is in the url in `linkGoogleAcctMocked`, but when we GET it, there is no `location` in the response headers.
actually i don't understand why we GET so many times (https://github.com/uc-cdis/gen3-qa/blob/master/services/apis/fence/fenceTasks.js#L219-L220), do you know why?

Reply:
wait, so you do see the error in the url? can we check for that here?

Reply:
fixed in 73ede732b60574df689d06e8a89137314038229e